### PR TITLE
adding udp

### PR DESCRIPTION
**Pull request by Guilhermeslucas**
Solving issue #2

### CODE REVIEW

File: knocker.py [hunk 1]
```diff
@@ -1,14 +1,17 @@
-import argparse
+iimport argparse
```
Comment (rougeth) [inline knocker.py]: typo

File: knocker.py [hunk 2]
```diff
@@ -26,8 +29,13 @@ def main():
     parser.add_argument('ports', nargs='+', type=int, help='Ports to knock')
     parser.add_argument('-d', '--delay', type=int, default=200,
                         help='Milliseconds between each knock')
+    parser.add_argument('-p', '--protocol', default='tcp')
 
     args = parser.parse_args()
 
+    if not(args.protocol.lower() == 'tcp' or args.protocol.lower() == 'udp'):
```
Comment (rougeth) [inline knocker.py]: A better way to check the options would be this:`if args.protocol.lower() not in ['tcp', 'udb']`. But, argparse has a choices options: https://docs.python.org/3/library/argparse.html#choices.

**Review by rougeth — CHANGES_REQUESTED** [review]
Sorry for the delay, completely forgot about this PR.

**Comment by Guilhermeslucas** [comment]
Done!

**Comment by rougeth** [comment]
Sorry @Guilhermeslucas, I wasn't clear enough. `argparser` has an option called `choices`, you could do something like:

```python
parser.add_argument('-p', '--protocol', choices=['tcp', 'udb'], default='tcp')
```

Got it?

**Comment by Guilhermeslucas** [comment]
Is this it?

File: knocker.py [hunk 3]
```diff
@@ -26,8 +29,13 @@ def main():
     parser.add_argument('ports', nargs='+', type=int, help='Ports to knock')
     parser.add_argument('-d', '--delay', type=int, default=200,
                         help='Milliseconds between each knock')
-
+    parser.add_argument('-p', '--protocol',choices=['tcp', 'udb'], default='tcp')
```
Comment (rougeth) [inline knocker.py]: There is a typo here `udp`.

File: knocker.py [hunk 4]
```diff
@@ -26,8 +29,13 @@ def main():
...
+    
     args = parser.parse_args()
 
+    if not(args.protocol.lower() in ['tcp','udp']):
+        print('Not a valid protocol. Use udp or tcp')
+        exit()
```
Comment (rougeth) [inline knocker.py]: You can remove this entire if, the `choices` argument already validate it.

**Review by rougeth — CHANGES_REQUESTED** [review]
Almost there, just those two comments and we're ready to merge.

**Review by rougeth — APPROVED** [review]

**Pull request merged**